ui/pages/swaps/prepare-swap-page/prepare-swap-page.js

```diff
@@ -1024,6 +1032,21 @@ export default function PrepareSwapPage({
               {selectedToToken?.string && yourTokenToBalance}
             </div>
           </Box>
+          <Box
+            display={DISPLAY.FLEX}
+            justifyContent={JustifyContent.spaceBetween}
+            alignItems={AlignItems.stretch}
+          >
+            <div className="prepare-swap-page__balance-message">
+              {selectedToToken &&
+                !isEmpty(selectedToToken) &&
+                isNonDefaultToToken &&
+                t('swapTokenVerifiedSources', [
+                  occurrences,
+                  <BlockExplorerLink key="block-explorer-link" />,
```

Comment:
The link to Zksync Explorer is broken for the [same reason](https://github.com/MetaMask/metamask-extension/issues/27476) as the link on the asset page is broken. 

https://github.com/user-attachments/assets/00b2728c-9aa2-4d92-81ea-05dc10090928


I think we need to update the `@metamask/etherscan-link` ([repo here](https://github.com/MetaMask/etherscan-link)) package to use a different link logic if the network is ZKSync.

Reply:
@jclancy93 @angelapotter , what would your thoughts be here? @martahj and I were thinking we could just hide the message for zkSync until that link repo is fixed, does that make sense to you guys as well?

Reply:
Instead of hiding it altogether, we could add a conditional [here](https://github.com/MetaMask/metamask-extension/blob/develop/ui/pages/swaps/prepare-swap-page/prepare-swap-page.js#L438) to use the getAccountLink instead of getTokenTracker link for the zksync network

I don't have a strong preference between hiding altogether or using the account link

Reply:
Great suggestion, @jclancy93! I went ahead with this solution, as it fixes the issue and is fairly straightforward